lib/seahorn/BvOpSem2ExtraWideMemMgr.hh

```diff
@@ -32,29 +29,22 @@ template <class T> class ExtraWideMemManager : public OpSemMemManagerBase {
 
   const Expr m_uninit_size;
 
-  static const unsigned int g_slotBitWidth = 64;
-  static const unsigned int g_slotByteWidth = g_slotBitWidth / 8;
-
-  static const unsigned int g_uninit = 0xDEADBEEF;
-  static const unsigned int g_uninit_small = 0xDEAD;
-  static const unsigned int g_num_slots = 3;
-
   /// \brief Memory manager for raw pointers
   T m_main;
   /// \brief Memory manager for pointer offset
 
   // NOTE: offset bitwidth is same as ptr bitwidth since we need to
   // do arithmetic operations between them often.
-  RawMemManager m_offset;
+  OpSemMemManagerMixin<RawMemManager> m_offset;
   /// \brief Memory manager for size
-  RawMemManager m_size;
+  OpSemMemManagerMixin<RawMemManager> m_size;
 
 public:
   using TrackingTag = typename T::TrackingTag;
-  using RawPtrTy = typename T::PtrTy;
-  using RawMemValTy = typename T::MemValTy;
-  using RawPtrSortTy = typename T::PtrSortTy;
-  using RawMemSortTy = typename T::MemSortTy;
+  using RawPtrTy = Expr;
```

Comment:
why these types degenerated to `Expr`?

Reply:
Fixed... was trying something out during refactoring